packages/appeals-service-api/src/lib/notify.js

```diff
@@ -0,0 +1,69 @@
+const { NotifyClient } = require('notifications-node-client');
+const fetch = require('node-fetch');
+const config = require('./config');
+const logger = require('./logger');
+const { getLpa } = require('../services/lpa.service');
+
+function getAddress(siteAddress) {
+  let address = '';
+  if (siteAddress.addressLine1) {
+    address = `${siteAddress.addressLine1}\n`;
+  }
+  if (siteAddress.addressLine2) {
+    address = `${address}${siteAddress.addressLine2}\n`;
+  }
+  if (siteAddress.town) {
+    address = `${address}${siteAddress.town}\n`;
+  }
+  if (siteAddress.county) {
+    address = `${address}${siteAddress.county}\n`;
+  }
+  if (siteAddress.postcode) {
+    address = `${address}${siteAddress.postcode}`;
+  }
+  return address;
+}
+
+async function sendEmail(appeal) {
+  logger.info(`Sending email...`);
+  logger.debug(`appeal = \n${JSON.stringify(appeal, null, 2)}`);
+  const { baseUrl, serviceId, apiKey, templateId } = config.services.notify;
+  const args = [];
+  if (baseUrl) {
+    args.push(baseUrl, serviceId);
```

Comment:
If `baseUrl` is optional, I'll change my config to make it so

Reply:
This taken from the pattern used in the existing mock notify service. If these values are provided then the mock is used. If not present the default 'real' Notify service will be used.